superset/cli/thumbnails.py

```diff
@@ -94,13 +93,7 @@ def compute_generic_thumbnail(
                 action = "Processing"
             msg = f'{action} {friendly_type} "{model}" ({i+1}/{count})'
             click.secho(msg, fg="green")
-            if friendly_type == "chart":
-                url = get_url_path(
-                    "Superset.slice", slice_id=model.id, standalone="true"
-                )
-            else:
-                url = get_url_path("Superset.dashboard", dashboard_id_or_slug=model.id)
-            func(url, model.digest, force=force)
+            func(None, model.id, force=force)
```

Comment:
the cache thumbnail functions here take a user id, model Id and optional force arguments. I'm not sure why we were passing url and digest here, but I wrote a test below that asserted that we should be passing the correct args. This might be worth testing to see how this was working before.